mindmeld/query_factory.py

```diff
@@ -141,6 +151,15 @@ def create_query_factory(
         language, locale = get_language_config(app_path)
         tokenizer = tokenizer or Tokenizer.create_tokenizer()
         stemmer = stemmer or get_language_stemmer(language_code=language)
+        if app_path:
+            sys_recognizer = SystemEntityRecognizer.get_instance(app_path)
```

Comment:
here as well.

Reply:
`app_path` is an optional argument for `create_query_factory`, so we need a default system entity recognizer.